### ISSUE TITLE
Ciphertext on the first message

### ISSUE THREAD

**Issue by asdasdsdasdasdas** [open-wa/wa-automate-nodejs#2278]
### Are you using the latest version of the library?

- [X] I have checked and am using the latest version of the library.

### Current Behavior

When someone contact the bot for the first time the type of message shows as "ciphertext" and the object "message" don't contein the value "body" so, it doesn't answer and the proyect crashes because body is not defined.

### Expected Behavior

Detect the body on the first message

### Steps To Reproduce

![2021-11-08_14h34_52](https://user-images.githubusercontent.com/54508766/140823897-3fd4c510-35b9-4370-a9e7-e70e56c5945c.png)
![2021-11-08_14h36_16](https://user-images.githubusercontent.com/54508766/140823911-350763c8-adb4-4525-bb64-5cdb5308d704.png)

### Mode

EASY API/CLI

### create() code

```markdown
wa.create({
  multiDevice: true,
  sessionId: "ChatbotSession",
  killProcessOnBrowserClose: true,
  qrTimeout: 0,
  authTimeout: 0,
}).then(client => start(client));
```

### DEBUG INFO

```markdown
Debug Info {
  WA_VERSION: 2.2142.12,
  PAGE_UA: WhatsApp/2.2108.8 Mozilla/5.0 (Macintosh; Intel Mac OS X 11_2_2) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/89.0.4389.90 Safari/537.36,
  WA_AUTOMATE_VERSION: 4.23.7,
  BROWSER_VERSION: HeadlessChrome/93.0.4577.0
}
```

### Environment

```markdown
- OS: 20.04 (LTS) x64
- Node: 16.8.0
- npm: 8.1.0
```

**Comment by asdasdsdasdasdas** [comment]
Update: the problem comes from the multi device because without multi-device don't have any error with the first message

**Comment by daopapp** [comment]
For me, occurs on first message on MD the type unknow and subtype fanout

<img width="529" alt="Captura de Tela 2021-11-09 às 09 55 29" src="https://user-images.githubusercontent.com/42589779/140928292-2a47c8f6-b349-425d-a138-ba0d9f36e5b0.png">

**Comment by smashah** [comment]
@asdasdsdasdasdas @daopapp I have tried to implement a fix for this but i'm kinda flying blind because I'm not able to consistently reproduce this issue. But I think I've found a fix.

1. You will still receive this ciphertext message with an missing body
2. The body will not return `undefined`, at worst it will return `""` which will prevent the `undefined` error 
3. You might receive the actual message soon after (let me know) - you may not receive the message if in your web session the message looks like this:

![image](https://user-images.githubusercontent.com/5903821/141235942-dc14f698-f353-4036-bfc6-dfc5f291b18e.png)


Due to this screenshot I was not able to resolve the message. It seems to be a security/privacy feature. 

Basically, despite this fix, you will still have circumstances where the ciphertext message doesn't eventually emit the actual message - so I suggest never relying on the first incoming message to do anything.

**Comment by daopapp** [comment]
Hi man, 
The problem it's that we use the first message as input for an automation. So, on this case, i've got the opt-in of client and depend of message content, we send a different response to client.

**Comment by daopapp** [comment]
No problem man....
The strange it's that the type unknow and the subtype fanout only occours on MD.
In chypertext i use the getMessageById to try to get the content of message body.
I think, that the problem it's when the user has a old dated whatsapp version and send message to MD...

**Comment by smashah** [comment]
@daopapp yeah I noticed that about your screenshot, however, I was only to get a `ciphertext/fanout` message, no `unknown/fanout` message. Fanout is a consistent theme with MD as a message (of any type) needs to ["Fan out"](https://en.wikipedia.org/wiki/Fan-out_(software)#Message-oriented_middleware) messages to multiple recipients (each MD session and the host account device session count as different recipients technically).

I think this issue will be finally resolved after the transition to MD is complete. So I will be closing this for now. Thanks

**Comment by asdasdsdasdasdas** [comment]
I should clarify that this problem of the ciphertext in the first message that the bot receives for the first time from a number only happens using Multi-device.
Without Multi-device the body of the message does appear but curiously the name of the contact does not, on the other hand in Multi-device the name of the contact does appear but not the body of the message.

You can generate and see these errors with console.log(message) and wait for them to send you a message from a number which has never contacted their chatbot
